hl_pitch.md

```diff
@@ -0,0 +1,28 @@
+# Human learning pitch
+
+## Preparation
+
+Do the path in the application at least once before to see if anything crashes.
+
+Predownload an image if the demo is done on an emulator, then add this image during the demo instead of taking one.
+
+## Script
+
+Verbal communication is central to human interaction. This means that if you can't communicate verbally, you cannot integrate well into society. People with mental disabilities can have problems communicating verbally, which is why they often use picture based communication.
+
+The problem now is that the same people do not have good abstraction capabilites. If they show you a picture of a fork and you give them a fork that does not look like the fork in the picture they do not recognize it as being what they requested.
+
+A method to teach this abstraction is picture matching, which would traditionally require printing lots of images. With our app you can do all of this digitally, which greatly simplifies the process.
+So let's learn!
+
+I have a tablet here with a big screen so I can see the images better. Let's see what datasets are available to me...
+
+_goes and looks at dataset overview_
+
+But there are only very few images... I want to have more objects to match! This is why I am going to go and augment my dataset.
+
+_adds picture and shows it added afterwards_
+
+Now let's learn some abstraction!
+
+_goes and learns in representation mode_
```

Comment:
Shouldn't we put the learning before the editing, to first showcase the main feature of the app?

Reply:
I decided the take the approach "first let's look what I can learn on" but one could argue that "I want to learn, oh my experience was not that good let's add more images" might actually make more sense

Reply:
my iteration of the demo was criticised with this exact angle